Source/WebCore/PAL/pal/crypto/openssl/CryptoDigestOpenSSL.cpp

```diff
@@ -154,5 +154,4 @@ std::optional<Vector<uint8_t>> CryptoDigest::computeHash(CryptoDigest::Algorithm
     digest->m_context->addBytes(data.data(), data.size());
     return digest->m_context->computeHash();
 }
-
```

Comment:
nit: probably don't need to delete this whitespace line, especially since you are not making any code changes to this file.

Reply:
removed.